dashboards/cluster.json

```
@@ -250,14 +265,14 @@
             "text": "All",
             "value": "$__all"
           },
-          "definition": "label_values(target_info{k8s_cluster_name=\"$cluster\"},k8s_namespace_name)",
+          "definition": "label_values(k8s_namespace_phase{k8s_cluster_name=\"$cluster\"},k8s_namespace_name)",
```

`$cluster` has multi select enabled

```suggestion
          "definition": "label_values(k8s_namespace_phase{k8s_cluster_name=~\"${cluster:pipe}\"},k8s_namespace_name)",
```